lib/jake.ex

```diff
@@ -11,46 +11,89 @@ defmodule Jake do
     "string"
   ]
 
-  def gen(%{"anyOf" => options} = spec) when is_list(options) do
+  def generator(map) do
+    StreamData.sized(fn size ->
+      Map.put(%{}, "map", map) |> Map.put("omap", map) |> Map.put("size", size) |> gen_init()
+    end)
+  end
+
+  def gen_init(schema) do
```

Comment:
probably gen_lazy might be a better name?

Reply:
ok, seems better.